lightningd/channel.h

```diff
@@ -64,6 +64,9 @@ struct channel {
 	/* Channel if locked locally. */
 	struct short_channel_id *scid;
 
+	/* Closing txid, for reporting to JSON */
+	struct bitcoin_txid *closing_txid;
```

Comment:
This field will not be repopulated when restarting / loading from DB, could that lead us into an inconsistent behavior? Before restart we show the `closing_txid` after the restart we lose that. If we broadcast the transaction we'll have it in the `transactions` table btw.